lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1350 @@
+module: podman_container
+author:
+  - "Sagi Shnaidman (@sshnaidm)"
+version_added: '2.9'
+short_description: Manage podman containers
+notes: []
+description:
+  - Start, stop, restart and manage Podman containers
+requirements:
+  - "Podman installed on host"
+options:
+  name:
+    description:
+      - Name of the container
+    required: True
+    type: str
+  executable:
+    description:
+      - Path to C(podman) executable if it is not in the C($PATH) on the machine running C(podman)
+    default: 'podman'
+    type: str
+  state:
+    description:
+      - I(absent) - A container matching the specified name will be stopped and removed.
+      - I(present) - Asserts the existence of a container matching the name and any provided configuration parameters. If no
+          container matches the name, a container will be created. If a container matches the name but the provided configuration
+          does not match, the container will be updated, if it can be. If it cannot be updated, it will be removed and re-created
+          with the requested config. Image version will be taken into account when comparing configuration. Use the recreate option
+          to force the re-creation of the matching container.
+      - I(started) - Asserts there is a running container matching the name and any provided configuration. If no container
+          matches the name, a container will be created and started. Use recreate to always re-create a matching container, even
+          if it is running. Use force_restart to force a matching container to be stopped and restarted.
+      - I(stopped) - Asserts that the container is first I(present), and then if the container is running moves it to a stopped
+          state.
+    type: str
+    default: 'started'
+    choices:
+      - absent
+      - present
+      - stopped
+      - started
+  image:
+    description:
+      - Repository path (or image name) and tag used to create the container. If an image is not found, the image
+        will be pulled from the registry. If no tag is included, C(latest) will be used.
+      - Can also be an image ID. If this is the case, the image is assumed to be available locally.
+    type: str
+  annotation:
+    description:
+      - Add an annotation to the container. The format is key value, multiple times.
+    type: dict
+  authfile:
+    description:
+      - Path of the authentication file. Default is ``${XDG_RUNTIME_DIR}/containers/auth.json``
+        (Not available for remote commands) You can also override the default path of the authentication file
+        by setting the ``REGISTRY_AUTH_FILE`` environment variable. ``export REGISTRY_AUTH_FILE=path``
+    type: str
+  blkio_weight:
+    description:
+      - Block IO weight (relative weight) accepts a weight value between 10 and 1000
+    type: int
+  blkio_weight_device:
+    description:
+      - Block IO weight (relative device weight, format DEVICE_NAME[:]WEIGHT).
+    type: str
```

Comment:
Is there precisely one time you can specify this? It sounds to me like this should be allowed to repeat multiple times. So this should be a `list`. Or maybe even better a `dict`, so people can specify things like
```
blkio_weight_device:
   /dev/sda: 100
   /dev/sdb: 200
```